Lib/email/charset.py

```diff
@@ -16,6 +16,7 @@
 
 from email import errors
 from email.encoders import encode_7or8bit
+from encodings.aliases import aliases as ALIASES
```

Comment:
The aliases used in the encoding package map to module names, not widely used common encoding names.